pkg/broker/config/config.go

```diff
@@ -29,11 +25,9 @@ func NewOptions(ctx *broker.Context) Options {
 
 // Options represents all the available options
 type Options struct {
-	Ctx                   *broker.Context
-	Constructor           Constructor
 	Codec                 codec.Constructors
 	Callers               transport.Callers
-	Listeners             transport.Listeners
+	Listeners             transport.ListenerList
```

Comment:
Question: Why did you not like `transport.Listeners`?

Reply:
I have been using the term `...List` in other packages such as the `specs`. I thought it was appropriate to rename this type as well